packages/sanity/src/_internal/cli/commands/schema/extractSchemaCommand.ts

```diff
@@ -0,0 +1,31 @@
+import {type CliCommandDefinition} from '@sanity/cli'
+
+const description = 'Extracts a json representation of a sanity schema within a Studio context.'
+
+const helpText = `
+**Note**: This command is experimental and subject to change.
+
+Options
+  --workspace <name> The name of the workspace to generate a schema for
+  --path Optional path to specify destination of the schema file
+  --enforce-required-fields Makes the schema generated respect required fields in the schema. This can be used when generating types if you are only querying for published documents. Defaults to false.
+
+Examples
+  # Extracts schema types in a Sanity project with more than one workspace
+  sanity schema extract --workspace default
+`
+
+const extractSchemaCommand: CliCommandDefinition = {
+  name: 'extract',
+  group: 'schema',
+  signature: '',
+  description,
+  helpText,
```

Comment:
Since this is an experimental/beta feature, maybe we should hide it from the help commands output for now?
```suggestion
  helpText,
  hideFromHelp: true,
```